ci_build_images/jepsen-mariadb.Dockerfile

```diff
@@ -38,8 +38,8 @@ USER buildbot
 
 WORKDIR /home/buildbot
 
-# Clone the jepsen-mysql repository, download leiningen, and set permissions
-RUN git clone https://github.com/vlad-lesin/jepsen-mysql jepsen-mariadb \
+# Clone the jepsen-mariadb repository, download leiningen, and set permissions
+RUN git clone https://github.com/MariaDB/jepsen-mariadb jepsen-mariadb \
```

Comment:
This pins the scripts to the repository version at the time of image build.
Is this the desired behavior?

If not, at least a Pull should happen during a BB-Build or git clone to be moved out of dockerfile.

Reply:
I understood that's the desired behaviour, but we can add a git pull in a step if it needs to be updated. The files are needed during image creation